ksa_compliance/hooks.py

```diff
@@ -134,7 +134,10 @@
 doc_events = {
     "Sales Invoice": {
         "on_submit": "ksa_compliance.standard_doctypes.sales_invoice.create_sales_invoice_additional_fields_doctype",
-        "before_save": "ksa_compliance.standard_doctypes.sales_invoice.calculate_tax_amount",
+        "before_save": [
+            "ksa_compliance.standard_doctypes.sales_invoice.calculate_tax_amount",
+            "ksa_compliance.standard_doctypes.sales_invoice.validate_tax_category"
```

Comment:
Validations should go into the validate hook, not before_save.

Reply:
done